src/pages/preferences/preferencesTabs/notificationTab/UserSubscriptions.tsx

```diff
@@ -0,0 +1,151 @@
+import { useMemo, useRef } from "react";
+import { useQuery } from "@apollo/client";
+import styled from "@emotion/styled";
+import {
+  Cell,
+  ExpandedContent,
+  flexRender,
+  HeaderCell,
+  HeaderGroup,
+  HeaderRow,
+  LeafyGreenTableRow,
+  Row,
+  Table,
+  TableBody,
+  TableHead,
+  useLeafyGreenTable,
+} from "@leafygreen-ui/table/new";
+import { SettingsCard, SettingsCardTitle } from "components/SettingsCard";
+import { StyledRouterLink } from "components/styles";
+import { getSubscriberText } from "constants/subscription";
+import {
+  GeneralSubscription,
+  Selector,
+  UserSubscriptionsQuery,
+  UserSubscriptionsQueryVariables,
+} from "gql/generated/types";
+import { USER_SUBSCRIPTIONS } from "gql/queries";
+import { notificationMethodToCopy } from "types/subscription";
+import { resourceTypeToCopy, triggerToCopy } from "types/triggers";
+import { getResourceRoute, processSubscriptionData } from "./utils";
+
+export const UserSubscriptions: React.VFC<{}> = () => {
+  const { data } = useQuery<
+    UserSubscriptionsQuery,
+    UserSubscriptionsQueryVariables
+  >(USER_SUBSCRIPTIONS);
+
+  const subscriptions = useMemo(
```

Comment:
I noticed that personal project subscriptions don't appear to be showing up in the table. Is it possible they are missing?

example project subscription I have on legacy UI:

<img width="357" alt="Screenshot 2023-05-19 at 12 47 36 PM" src="https://github.com/evergreen-ci/spruce/assets/47064971/d91f8d56-c596-4ca0-bcf1-c4ef7144ab3f">

Reply:
Great catch! Based on how the data is actually structured projects look like this instead of how they do in the designs, if that's okay. I think it's still quite clear what is being monitored!
<img width="1085" alt="image" src="https://github.com/evergreen-ci/spruce/assets/9298431/59baf639-28bf-4bd5-b726-759703c29bd7">